helm/oauth2-proxy/templates/deployment.yaml

```
@@ -160,6 +160,11 @@ spec:
               name:  {{ template "oauth2-proxy.secretName" . }}
               key: cookie-secret
         {{- end }}
+        envFrom:
+        {{- with .Values.proxyVarsFromSecret }}  
+        - secretRef:
+            name: {{  . }}
+        {{- end }}
```

```suggestion
        {{- with .Values.proxyVarsFromSecret }}  
        envFrom:
        - secretRef:
            name: {{  . }}
        {{- end }}
```
To be moved before line 145.